patches/minecraft/net/minecraft/entity/player/PlayerEntity.java.patch

```diff
@@ -313,6 +317,14 @@
 +   public Collection<IFormattableTextComponent> getSuffixes() {
 +       return this.suffixes;
 +   }
++   
++   private ITextComponent displayname = null;
++   /**
++    * Force the displayed name to refresh
++    */
```

Comment:
That JavaDoc explains almost nothing. You should at least link to the event name and clarify that this will cause the event to be re-fired.

Reply:
Fair point